### DISCUSSION TITLE
ShoppyGo: an e-commerce multi vendor platform based on PrestaShop

### DISCUSSION THREAD

**Discussion by luigimassa** [PrestaShop/PrestaShop#32183] [Showcase]
Hi everyone! 👋 We want to introduce you to ShoppyGo.io, the ideal platform for those who want to create and manage a successful Multi Vendor E-Commerce, based on PrestaShop.

🌟 Awesome features to manage your Multi Vendor E-Commerce 🌟
ShoppyGo offers all the features you need and more. From managing sellers and products, to customizing the user interface and shopping experience, with ShoppyGo you'll have everything under control.

ShoppyGo is not a module, but a new **bundle** integrated in the core of PrestaShop.
[This is the repo](https://github.com/ShoppyGo)
[https://www.shoppygo.io](https://www.shoppygo.io)

**Comment by matks** [comment]
Hi @luigimassa it's cool to see a nice usage of Symfony bundles inside PrestaShop ^^

If you have encountered technical difficulties while doing that, we'd be happy to hear about it.

Makes me think about https://github.com/PrestaShop/PrestaShop/issues/27188 which is a problem we have not yet found a solution for so far

**Reply by luigimassa** [reply]
Hello @matks. 
I just wanted to share a showcase, a PrestaShop Flavour: PrestaShop transformed in Marketplace with an integrated bundle and not with a module
Thanks anyway for the suggestions
